paper/paper.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "CDP: Software for Municipal Data Collection, Analysis, and Publication"
+tags:
+    - Python
+    - JavaScript
+    - open government
+    - open data
+    - open infrastructure
+    - municipal governance
+    - data archival
+    - civic technology
+    - natural language processing
+authors:
+    - name: Jackson Maxfield Brown
+      orcid: 0000-0003-2564-0373
+      affiliation: 1
+    - name: To Huynh
+      orcid: 0000-0002-9664-3662
+      affiliation: 2
+    - name: Isaac Na
+      orcid: 0000-0002-0182-1615
+      affiliation: 3
+    - name: Brian Ledbetter
+      affiliation: 1
+    - name: Hawk Ticehurst
+      affiliation: 1
+    - name: Sara Liu
+      affiliation: 4
+    - name: Emily Giles
+      affiliation: 4
+    - name: Katlyn Greene
+      affiliation: 4
+    - name: Sung Cho
+      affiliation: 4
+    - name: Shak Ragoler
+      affiliation: 4
+    - name: Nicholas Weber
+      orcid: 0000-0002-6008-3763
+      affiliation: 1
+
+affiliations:
+    - name: University of Washington iSchool, University of Washington, Seattle
+      index: 1
+    - name: University of Washington, Seattle
+      index: 2
+    - name: McKelvey School of Engineering, Washington University, St. Louis
```

Comment:
Could this be changed to just `Washington University in St. Louis`?

Reply:
Done!